concreteproperties/design_codes.py

```diff
@@ -287,17 +291,23 @@ def create_concrete_material(
         compressive_strength: float,
         colour: str = "lightgrey",
     ) -> Concrete:
-        r"""Returns a concrete material object to AS 3600:2018.
-
-        | **Material assumptions:**
-        | - *Density*: 2400 kg/m\ :sup:`3`
-        | - *Elastic modulus*: Interpolated from Table 3.1.2
-        | - *Service stress-strain profile*: Linear with no tension, compressive strength
-          at 0.9 * f'c
-        | - *Ultimate stress-strain profile*: Rectangular stress block, parameters from
-          Cl. 8.1.3
-        | - *Alpha squash*: From Cl. 10.6.2.2
-        | - *Flexural tensile strength*: From Cl. 3.1.1.3
+        """Returns a concrete material object to AS 3600:2018.
+
+        .. admonition:: Material assumptions
```

Comment:
This is nice!

Reply:
I thought it stood out more in the docs.